assets/check

```diff
@@ -47,10 +48,18 @@ if [ -n "${ca_bundle}" ]; then
   export AWS_CA_BUNDLE=/tmp/ca.pem
 fi
 
+with_enc=""
+if [ ! "${encryption_key}" = "" ];then
+  echo "Encrypting with GnuPG..."
+  find "${source}/${path}" -type f -name "*${suffix}" -exec \
```

Comment:
Whare is $source comming from? I can't find any earlier definition.

Reply:
Pushed new commit fixing this